src/ffi/cpython/initconfig.rs

```diff
@@ -1,9 +1,21 @@
 /* --- PyStatus ----------------------------------------------- */
 
 use crate::ffi::Py_ssize_t;
-use libc::wchar_t;
 use std::os::raw::{c_char, c_int, c_ulong};
 
+#[cfg(feature = "libc")]
+use libc::wchar_t;
+
+#[cfg(not(feature = "libc"))]
+#[allow(non_camel_case_types)]
+#[cfg(target_family = "unix")]
+type wchar_t = i32;
```

Comment:
This doesn't appear to be the type used by all unix like platforms, for example:
https://github.com/rust-lang/libc/blob/1c66799b7b8b82269c6bff0eab97d1a30e37fd36/src/unix/linux_like/linux/gnu/b64/aarch64/mod.rs#L4

Reply:
Thanks for pointing that out, I can't say I've looked at that architecture before. Not sure there is enough interested to make changes though so I'll just close ;)

Reply:
Yeah, so it looks like the non-arch specific unix types are mostly here, so those are probably reasonably safe to use with this technique, but stuff like `wchar_t` appears to be more complex as you'd want to scope them to specific architectures like `target_arch = "x86_64"` or something:
https://github.com/rust-lang/libc/blob/0.2.97/src/unix/mod.rs